### ISSUE TITLE
1.19.3: timed.rs - Can it be used for spawning mobs?

### ISSUE THREAD

**Issue by huntingzap** [kangarko/ChatControl#2267]
### Are you using MySQL?

No

### Are you using BungeeCord/Velocity?

Yes - Velocity

### Question

Hello,
I was wanting to use the timed.rs file under the messages folder to run the following:

group global
then sound AMBIENT_CAVE, 1.0, 0.8
then bossbar red SOLID 10 &bA snow man boss has spawned in the end!
then console mm m kill SnowManBoss
then console mm m spawn SnowManBoss 1 world_the_end,-2.5,59,22.5,1,1
delay 21600 second
message:
- &f
&c&lBOSS &8» &fA snowman boss has spawned in the end arena! &7(/warp end)
&f

Basically, I want to use it to start a boss event. However, this only seems to work when an OP'd player is in the end world. If there is no player in the end it just doesnt seem to spawn anything. Is this because of MythicMobs, or am I missing something/ doing something wrong with this format. Everything works great, but only when an OP seems to be in the end. Any input would be great, thanks!

**Comment by TomLewis** [comment]
Why would you use ChatControl as your server scheduler? You can use;

1. Cron
2. Your control panel/wrapper scheduler
3. A scheduler built into something like CMI

**Comment by kangarko** [comment]
Yes it could but a little hacky, you could create a javascript variable in variables/ folder calling Bukkit.dispatchCommand or hooking into Boss API directly using JavaScript. It would return an empty "" string so it is not visible.

Then you could put this variable in your message somewhere like "A snowman has been spawned. {spawn_boss}"

A little hacky and dirty but could work.

**Comment by TheIntolerant** [comment]
![image](https://user-images.githubusercontent.com/48093184/214333811-9bbc335e-7986-4cc5-8861-6a9406d37e36.png)
It might also be a case of moving it from here. This is useful if you have multiple timed message groups and you want to display them all.

**Comment by kangarko** [comment]
That's up to people to remove it or I am missing something?

**Comment by TheIntolerant** [comment]
> That's up to people to remove it or I am missing something?

Nah, they said, 

"this only seems to work when an OP'd player is in the end world." 

So I figured, this might have something to do with it. Especially since there is nothing in this group, that would prevent it from working for a non-OP person. So I thought it was worth mentioning a possible reason why it isn't working for everyone else.

**Comment by kangarko** [comment]
Sorry I still seem not to understand, do we have a specific message that's meant to only work for OPs in the end world? The "timed" messages should work for everyone, anywhere.

**Comment by TheIntolerant** [comment]
So my thinking was that the reason it might only work sometimes is that the other times, the users are only getting one of the other timed messages groups because that was the first message group they matched with.

**Comment by kangarko** [comment]
Oh I see thanks for explaining :)
